templates/resources/vrack_ip.md.tmpl

```diff
@@ -20,6 +20,7 @@ The following arguments are supported:
 
 * `service_name` - (Required) The internal name of your vrack
 * `block` - (Required) Your IP block.
+* `region` - (Optional) The region where want to route your block to.
```

Comment:
what is the format expected of the region (something like `GRA`, `EU-WEST-GRA`, `eu-west-gra`, ...) ? I think it would be easier for users if there was an example on this line